tools/clang/include/clang/SPIRV/SpirvBuilder.h

```diff
@@ -770,6 +770,10 @@ class SpirvBuilder {
   /// the feature used at the given source location.
   inline void requireCapability(spv::Capability, SourceLocation loc = {});
 
+  /// \brief Adds the given capability to the module under construction due to
+  /// the feature used at the given source location.
```

Comment:
```suggestion
  /// \brief Adds the given capability to the module under construction.
```

Reply:
I fixed it too many cut and paste error I did not fix up.